src/client/MwWindow.ts

```diff
@@ -11,6 +11,24 @@ interface MediaWiki {
 	config: MwConfig;
 }
 
+interface Wikibase {
+	getLanguageNameByCode( languageCode: string ): string;
+}
+
+interface ULS {
```

Comment:
This and JQuery seem to be here prematurely. They _may_ creep in on us soon, but are not needed just yet. Fine to keep them if you are confident of the interface definitions though and that we'll need them.